scripts/examples/oracle/oracle.sh

```diff
@@ -9,20 +9,18 @@ elysd query oracle params
 elysd query oracle list-price-feeder
 elysd query oracle show-price-feeder $(elysd keys show -a alice --keyring-backend=test)
 elysd query oracle list-price
-elysd query oracle show-price BTC --source="binance" --timestamp=1680860921 
-elysd query oracle show-price BTC --source="binance"
+elysd query oracle show-price BTC --source="elys" --timestamp=1680860921 
+elysd query oracle show-price BTC --source="elys"
 elysd query oracle show-price BTC
 elysd query gov proposals
 
 elysd tx oracle request-band-price
 elysd tx oracle add-asset-info-proposal satoshi BTC BTC "" "" --title="title" --description="description" --deposit="10000000000000000000aelys" --from=alice --chain-id=elys --broadcast-mode=block --yes
 elysd tx oracle remove-asset-info-proposal satoshi --title="title" --description="description" --deposit="10000000000000000000aelys" --from=alice --chain-id=elys --broadcast-mode=block --yes
-elysd tx oracle set-asset-info satoshi BTC BTC "" "" --from=alice --chain-id=elys --broadcast-mode=block --yes
```

Comment:
@junkai121 can you update the `deposit` flags that uses `aelys` it should be `uelys` instead thanks

Reply:
@junkai121 this is a minor issue we can fix that in a later PR

Reply:
Sure - I will put it as next PR.